GCEm/__init__.py

```diff
@@ -7,9 +7,201 @@
     for new users.
 
 """
-from .gp_model import GPModel
-from .sampler import Sampler
+from .cube_wrapper import CubeWrapper
+from .data_processors import Recast, Whiten, Reshape as Reshaper, Flatten
+from .emulator import Emulator
 
 __author__ = "Duncan Watson-Parris"
-__version__ = "0.1.0"
-__status__ = "Stable"
+__version__ = "0.2.0"
+__status__ = "Dev"
+
+
+def gp_model(training_params, training_data, data_processors=None,
```

Comment:
Would it be possible to also have the `data_processors` be a string input? Given there are only a couple of options it would be easier that way?

Reply:
The kernels need initialising with the right shape params, but I think there's less of an arugment for the data processors since you just need initialise them, so it's only `data_processors=['Log']` instead of `data_processors=[Log()]`...

Reply:
True that it's less important than the kernels, but for the processors you do still have to import them...

```
from GCEm.data_processors import Whiten
model = gp_model(X_train, Y_train, data_processors=[Whiten()])
```

Reply:
@guydagan, help me out here